.github/workflows/pr_title_check.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Submitty PR Title Check'
+name: 'PR Title Check'
```

Comment:
What do you think about renaming this to "Documentation"?  We may want to add more PR documentation checks in the future, and this workflow would be a good place to put them.

Reply:
That sounds good!